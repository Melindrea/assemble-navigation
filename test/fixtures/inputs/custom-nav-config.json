{
  "assemble": {
    "options": {
      "navigation": {
        "top":{},
        "side": {},
        "foot":{}
      }
    }
  }
}