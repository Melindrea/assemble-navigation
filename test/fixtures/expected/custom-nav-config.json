{
  "assemble": {
    "options": {
      "pages": [],
      "navigation": {
        "top":{},
        "side": {},
        "foot":{}
      }
    }
  }
}